src/DIRAC/Core/Utilities/Graphs/GraphUtilities.py

```diff
@@ -366,8 +366,8 @@ def get_locator(self, dmin, dmax):
         locator = RRuleLocator(rrule, self.tz)
         locator.set_axis(self.axis)
 
-        locator.set_view_interval(*self.axis.get_view_interval())
-        locator.set_data_interval(*self.axis.get_data_interval())
+        locator.axis.set_view_interval(*self.axis.get_view_interval())
+        locator.axis.set_data_interval(*self.axis.get_data_interval())
```

Comment:
I feel like this is a no-op?

Reply:
that is my feeling too, but I have absolutely no clue what these things are doing underneath, and I certainly do not want to put my finger anywhere near the current plotter. So I thought that in the worse case scenario, it does nothing :D

Reply:
I would like to merge/propagate this one quickly (basically now) and this can be fix isn't necessary and can be done later on. I wait few minutes anyway.

Reply:
it is necessary, as it breaks pylint